components/backdrop/README.md

```diff
@@ -1,30 +1,32 @@
-# Backdrop
-
-## d2l-backdrop
+# Backdrop [d2l-backdrop]
 
 The `d2l-backdrop` element is a web component to display a semi-transparent backdrop behind a specified sibling element. It also hides elements other than the target from assistive technologies by applying `role="presentation"` and `aria-hidden="true"`.
 
+<!-- docs: demo live name:d2l-backdrop size:small -->
 ```html
 <script type="module">
+  import '@brightspace-ui/core/components/button/button.js';
   import '@brightspace-ui/core/components/backdrop/backdrop.js';
+  import '@brightspace-ui/core/components/switch/switch.js';
 </script>
 <style>
-  #target { position: relative; z-index: 1000; }
+  #target { position: relative; z-index: 1000; margin: 40px; }
 </style>
-
-<div id="target"><button>toggle backdrop</button></div>
-<d2l-backdrop for-target="target" shown></d2l-backdrop>
-```
-
-Set the visible state of the backdrop by using the `shown` attribute/property.
-
-```javascript
-button.addEventListener('click', () => {
-  backdrop.shown = !backdrop.shown;
-});
+<div>
+  <div id="target"><d2l-button primary>Toggle backdrop</d2l-button></div>
+  <d2l-backdrop for-target="target"></d2l-backdrop>
+</div>
+<span>Background content</span>
```

Comment:
I left this demo a pretty simple for now. The backdrop is difficult to see on a white background so I also added some background content to make it more obvious